docs/docs/cmd/onedrive/onedrive-list.md

```diff
@@ -19,3 +19,227 @@ Retrieves a list of OneDrive sites from the tenant.
+        "IBMode": null,
+        "IBSegments": [],
+        "IBSegmentsToAdd": null,
+        "IBSegmentsToRemove": null,
+        "IsGroupOwnerSiteAdmin": false,
+        "IsHubSite": false,
+        "IsTeamsChannelConnected": false,
+        "IsTeamsConnected": false,
+        "LastContentModifiedDate": "/Date(2023,4,22,10,47,36,867)/",
+        "Lcid": 1033,
+        "LimitedAccessFileType": 0,
+        "ListsShowHeaderAndNavigation": false,
+        "LockIssue": null,
+        "LockState": "Unlock",
+        "LoopDefaultSharingLinkRole": 0,
+        "LoopDefaultSharingLinkScope": 0,
+        "LoopOverrideSharingCapability": false,
+        "LoopSharingCapability": 0,
+        "MediaTranscription": 0,
+        "OverrideBlockUserInfoVisibility": 0,
+        "OverrideSharingCapability": false,
+        "OverrideTenantAnonymousLinkExpirationPolicy": false,
+        "OverrideTenantExternalUserExpirationPolicy": false,
+        "Owner": "john@contoso.onmicrosoft.com",
+        "OwnerEmail": null,
+        "OwnerLoginName": null,
+        "OwnerName": null,
+        "PWAEnabled": 1,
+        "ReadOnlyAccessPolicy": false,
+        "ReadOnlyForBlockDownloadPolicy": false,
+        "ReadOnlyForUnmanagedDevices": false,
+        "RelatedGroupId": "/Guid(00000000-0000-0000-0000-000000000000)/",
+        "RequestFilesLinkEnabled": false,
+        "RequestFilesLinkExpirationInDays": 0,
+        "RestrictedAccessControl": false,
+        "RestrictedAccessControlGroups": null,
+        "RestrictedAccessControlGroupsToAdd": null,
+        "RestrictedAccessControlGroupsToRemove": null,
+        "RestrictedToRegion": 3,
+        "SandboxedCodeActivationCapability": 0,
+        "SensitivityLabel": "/Guid(00000000-0000-0000-0000-000000000000)/",
+        "SensitivityLabel2": null,
+        "SetOwnerWithoutUpdatingSecondaryAdmin": false,
+        "SharingAllowedDomainList": null,
+        "SharingBlockedDomainList": null,
+        "SharingCapability": 0,
+        "SharingDomainRestrictionMode": 0,
+        "SharingLockDownCanBeCleared": false,
+        "SharingLockDownEnabled": false,
+        "ShowPeoplePickerSuggestionsForGuestUsers": false,
+        "SiteDefinedSharingCapability": 2,
+        "SocialBarOnSitePagesDisabled": false,
+        "Status": "Active",
+        "StorageMaximumLevel": 1048576,
+        "StorageQuotaType": null,
+        "StorageUsage": 99,
+        "StorageWarningLevel": 943718,
+        "TeamsChannelType": 0,
+        "Template": "SPSPERS#10",
+        "TimeZoneId": 13,
+        "Title": "John",
+        "TitleTranslations": null,
+        "Url": "https://contoso-my.sharepoint.com/personal/john_contoso_onmicrosoft_com",
+        "UserCodeMaximumLevel": 300,
+        "UserCodeWarningLevel": 200,
+        "WebsCount": 0
+      }
+    ]
+    ```
+
+=== "Text"
+
+    ```text
+    Title: John
+    Url  : https://contoso-my.sharepoint.com/personal/john_contoso_onmicrosoft_com
```

Comment:
Usually, list commands return multiple results, when this happens, the text output looks a bit different (notice the horizontal dashes under the titles):

![image](https://github.com/pnp/cli-microsoft365/assets/11723921/6537c1dc-86fb-4f6a-b1c9-9d1f1458265a)

I suggest that we format it this way here as well.